src/app/control.rs

```diff
@@ -272,7 +268,7 @@ pub mod resolve {
             let target = client::Target {
                 addr,
                 server_name: dst.identity.clone(),
-                log_ctx: ::logging::admin().client("control", dst.addr.clone()),
+                log_ctx: crate::logging::admin().client("control", dst.addr.clone()),
```

Comment:
IMO, this would look nicer if we imported the `logging` module here (and elsewhere), but TIOLI.